docs/source/rules.rst

```diff
@@ -12,7 +12,7 @@ for the rule itself, with all the other mechanics abstracted away.
 Specific Rules
 --------------
 
-.. automodule:: sqlfluff.core.rules.std
+.. automodule:: sqlfluff.core.rules
```

Comment:
I moved it from that module! In the future if we decide to add some rules as plugins, they'll show up in the documentation website :)